deploy/Datadog.Trace.ClrProfiler.WindowsInstaller/Files.Managed.Net461.wxs

```diff
@@ -20,11 +20,146 @@
               Source="$(var.ManagedDllPath)\net461\Datadog.Trace.dll"
               KeyPath="yes" Checksum="yes"/>
       </Component>
+      <Component Win64="$(var.Win64)">
```

Comment:
We don't need to deploy any of these. I tried `dotnet publish` on my machine and they are not copied to the output folder.

Reply:
They are copied via publish on my machine.

Reply:
Can you merge/rebase `master` and run this again? A lot of these files should now be gone.

Reply:
If you check latest, that is the case.

Reply:
Thanks! Resolving this comment.